### PR TITLE
Ignore vendored files in coverage

### LINKED ISSUE #39: Fix coveragerc to not try to cover things that we don't need to check
Our coverage is artificially low, we can fix coveragerc to exclude anything that isn't under  (e.g. everything that isn't in `propkatraj.py` or `propkatraj/tests`.

We can also quickly add an nbval test (alongside the standard CI test action) to run the examples through - although that's not a coverage thing.

### PR DESCRIPTION
**Pull request by IAlibay**
Fixes #39

### CODE REVIEW

**Comment by IAlibay** [comment]
This should be good to go, @orbeckst you might need to add the codecov token afterwards if things start failing (I can't seem to be able to access the settings, but it might be that we need to merge this first).

**Review by ianmkenney — APPROVED** [review]
LGTM

File: .github/workflows/gh-ci.yaml [hunk 1]
```diff
@@ -79,16 +79,25 @@ jobs:
...
         pip list
         conda info
         conda list
+
     - name: Run tests
       run: |
-        pytest -v --color=yes propkatraj/tests/
+        pytest -n 2 --cov=propkatraj --cov-report=xml -v --color=yes propkatraj/tests/
+
+    - name: codecov
```
Comment (orbeckst) [inline .github/workflows/gh-ci.yaml:95]: If you want to use a toke, use a line such as https://github.com/alchemistry/alchemlyb/blob/045f85d5019d146eb1e7c14ba817e8e4ee1adfa3/.github/workflows/ci.yaml#L73

But let's use it in "we trust Github mode for right now" (just as @IAlibay said)

**Review by orbeckst — APPROVED** [review]
All good.

We can do codecov toke when necessary.

**Comment by orbeckst** [comment]
@IAlibay I changed your repo access to admin so you should be able to access settings. I didn't know that this wasn't included in maintainer.

**Pull request merged**